third_party/suitesparse.BUILD

```diff
@@ -0,0 +1,600 @@
+"""
+suitesparse@7.1.0
+"""
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "suitesparseconfig",
+    srcs = [
+        "SuiteSparse_config/SuiteSparse_config.c",
+    ],
+    hdrs = [
+        "SuiteSparse_config/SuiteSparse_config.h",
+    ],
+    includes = [
+        "SuiteSparse_config/",
+    ],
+    strip_include_prefix = "SuiteSparse_config",
+    linkopts = [
+        "-lm",
+        "-lpthread",
+        "-lblas",
```

Comment:
forewarning - for targeting arm we build against a sysroot so these packages might need to be added there

Reply:
I'm afraid I don't totally understand what that implies, either in general or for this PR.